magefile.go

```diff
@@ -844,6 +848,23 @@ func (Cloud) Push() error {
 	return nil
 }
 
+// Creates a new devmachine that will be auto-deleted in 6 hours.
+// Example: mage devmachine:create "pavel-dev-machine" "family/platform-ingest-elastic-agent-ubuntu-2204" "us-central1-a"
+func (Devmachine) Create(instanceName string, imageName string, zone string) error {
```

Comment:
Do we have a way of enumerating what images are available?

Reply:
Good question. I can implement it, but there are 2 ways:

- a simple gcloud command where you need to specify your email
- I can wrap it in a mage target like `mage devmachine:list`

What's would be better? 🤔

Reply:
Honestly, I think the best user experience would be to provide a reasonable default here (the ubuntu 22.04 one sounds fine to me), and if someone wants a different image, they can run the gcloud command. We should include that command in the help text though.

Reply:
We have a lot of different prebuilt images. `family/platform-ingest-elastic-agent-ubuntu-2204` stands for Ubuntu 22.04. 
Unfortunately, mage doesn't support default parameter values. But it can be achieved with env vars. For example, 
```
MACHINE_IMAGE="family/platform-ingest-elastic-agent-ubuntu-2204" ZONE="us-central1-a" mage devmachine:create "machine_name"
```
WDYT?

Reply:
The full list of available platform-ingest images is [here](https://docs.elastic.dev/ingest-dev-docs/engineer-productivity/custom-ci-images#available-vm-images). The image is specified in the [pipeline step definition](https://github.com/elastic/elastic-agent/blob/main/.buildkite/bk.integration.pipeline.yml#L52). It is better explained in the [Full doc](https://github.com/elastic/ingest-dev/pull/4248/files#diff-a95e8cfcce5a66414ad9917e12cadcbff18d2feff2a92da35a76367900cb2c8fR13) mentioned in the PR description

Reply:
```
MACHINE_IMAGE="family/platform-ingest-elastic-agent-ubuntu-2204" ZONE="us-central1-a" mage devmachine:create "machine_name"
```
Looks good to me.

Reply:
Now we specify `ZONE` and `MACHINE_IMAGES` as optional env vars. For example, `ZONE="europe-west1-b" mage -v devmachine:create "pavel-dev-machine"` will run a machine from `family/platform-ingest-elastic-agent-ubuntu-2204` image in `europe-west1-b` zone.

Reply:
Thanks, Mikołaj!